README.md

```diff
@@ -1,24 +1,157 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* How to run the test suite
+<!-- PROJECT DESCRIPTION -->
 
-* Services (job queues, cache servers, search engines, etc.)
+# 📖 [Blog App] <a name="about-project"></a>
 
-* Deployment instructions
+**[Bolg App]** is a simple Ruby on Rails CRUD app that allows the user to post, edit, save, and view their blog posts with time-stamped entries.
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- comming soon ⌛
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/) on your local machine
+- [Install Ruby on Rails](https://guides.rubyonrails.org/v5.1/getting_started.html)
+
+### Setup
+
+Clone this repository to your desired folder:
+- Clone [this repository](https://github.com/muskan2532150/blog-app.git)
+
+
+### Install
+
+Install this project with:
+  ```
+  gem install bundler
+  ```
+  Then run:
+  ```
+  bundle install
+  ```
+### Usage
+
+To run the project, execute the following command:
+
+  ```
+  rails s
+  ```
+  Open browser with `localhost:3000`
+
+### Run tests
+
+To run tests, run the following command:
+
+- rspec .
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Muskan Gupta**
+
+- GitHub: [@Muskan]()
+- Twitter: [@Muskan]()
+- LinkedIn: [@Muskan]()
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
```

Comment:
Great job @muskan2532150 🤝

- [ ] [Optional] Kindly fix the broken links in your readme file to make it more professional. You may fix the `author`, `issues page`, and the `MIT license` links.